tari-project/universe
# fix(ui): hide chip if c/gpu enabled | trigger interval fetch on mode change

Description
---

- only display CPU/GPU usage chip if that mining type is enabled and mining
- check for est. earnings balance so we don't render `0`
- ~~updated `useMiningStatesSync` to check for mining mode changes so the interval items are changed sooner than waiting for the interval - probably a MINOR improvement if any, as it's just a 1 second interval. i couldn't reproduce point two on the issue (neither are instant for me:p) but thought i'd add it in case it may help)~~ reverted this since it's such a quick interval

Motivation and Context
---
- resolves the first part of https://github.com/tari-project/universe/issues/842

How Has This Been Tested?
---
- locally

<img width="554" alt="image" src="https://github.com/user-attachments/assets/27b59bff-42c7-4cb7-82f3-41a41c4a4d59">


What process can a PR reviewer use to test or verify this change?
---
- switch mining modes with GPU mining only
- toggle GPU/CPU mining and see whether the usage chips are still there or not
